packages/webpack-cli/lib/utils/logger.js

```diff
@@ -1,3 +1,8 @@
-const { getLogger } = require('webpack/lib/logging/runtime');
+const { red, cyan, yellow } = require('colorette');
 
-module.exports = getLogger('webpack-cli');
+module.exports = {
+    error: (val) => console.error(`[webpack-cli] ${red(val)}`),
+    warn: (val) => console.warn(`[webpack-cli] ${yellow(val)}`),
+    info: (val) => console.info(`[webpack-cli] ${cyan(val)}`),
+    log: (val) => console.log(val),
```

Comment:
Can we add `[webpack-cli]` here too?

Reply:
There are some places we might not want to print `[webpack-cli]` for example with `--help`, `--version`. 
![Screenshot at 2020-08-12 16-26-35](https://user-images.githubusercontent.com/46647141/90007619-b101cd80-dcb8-11ea-9a07-5651836e1802.png)

Reply:
Let's implement `logger.log` with `[webpack-cli]` and `logger.help` without `webpack-cli` and using `logger.help` for `help`